CMakeLists.txt

```diff
@@ -232,9 +231,7 @@ else()
 endif()
 
 
-if(DETECT_PKG_CONFIG)
-  find_package(PkgConfig)
-endif()
+find_package(PkgConfig)
```

Comment:
@rcurtin Mainly due to the call in https://github.com/SoapGentoo/armadillo-code/blob/dbb8e02b6976faf20e82e97c05ec6c49af093f1f/CMakeLists.txt#L253
In general `FindHDF5.cmake` should take care of finding HDF5, so I don't think finding pkgconfig is a requirement for finding HDF5. Your take? I can totally nix it + the `pkg_check_modules(HDF5 hdf5)` call.